ADOPTERS.md

```diff
@@ -0,0 +1,2 @@
+* Everquote
```

Comment:
include Opstrace as well?

Reply:
Did they agree? The more the merrier! Will merge this one and if you know they agreed, toss them in right after! :)